roles/bootstrap-os/tasks/bootstrap-debian.yml

```diff
@@ -1,11 +1,11 @@
 ---
-#  raw: cat /etc/issue.net | grep '{{ bootstrap_versions }}'
-
-- name: Bootstrap | Check if bootstrap is needed
+- name: Check if bootstrap is needed
   raw: which "{{ item }}"
   register: need_bootstrap
   failed_when: false
   changed_when: false
+  # This command should also be called in check mode
```

Comment:
The comment and the value don't match. `check_mode` should be `true` if this is to always run even in check mode

Reply:
Uhm no.

If `check_mode: true` is set there, the module would _always_ think it is being called in check mode.

What I want is that the module always think it is NOT in check mode, even if the playbook is called that way. It looks a bit counter-intuitive, that's why I added the comment. Maybe I should be even more explicit and add an explanation like here, but in the end I don't want to document how Ansible works...

Reply:
Then please clarify the comment as it reads ambiguously to me.  Something like `This task should always run, even when check mode is enabled` or just remove it.

I do understand how ansible works ;), and as I said the comment and variable did not appear to match. I took the phrasing of your comment to mean you intended to always run the task in check mode. The "called in" part through me off, so I just needed your input on the intended behavior to know which (var vs comment) should be fixed.

Reply:
Hm, is `# This command should always run, even in check mode` better?